src/cryptoadvance/specter/devices/jade.py

```diff
@@ -0,0 +1,32 @@
+from .hwi_device import HWIDevice
+from .hwi.jade import JadeClient, enumerate
+
+
+class Jade(HWIDevice):
+    device_type = "jade"
+    name = "Jade"
+    icon = "jade_icon.svg"
+
+    supports_hwi_toggle_passphrase = True
+    supports_hwi_multisig_display_address = True
+
+    def __init__(self, name, alias, keys, blinding_key, fullpath, manager):
+        HWIDevice.__init__(self, name, alias, keys, blinding_key, fullpath, manager)
```

Comment:
If you are using exactly the same function as the parent class you can remove the whole `__init__` function declaration